hack/update-build.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+echo "Updating container file"
+
+: "${CONTAINER_FILE:=./Dockerfile}"
+: "${COMMIT:=$(git rev-list --abbrev-commit --tags --max-count=1)}"
+
+cat <<EOF >>"${CONTAINER_FILE}"
+LABEL com.redhat.component="network-observability-ebpf-agent-container"
+LABEL name="network-observability-ebpf-agent"
+LABEL io.k8s.display-name="Network Observability EBPF Agent"
+LABEL io.k8s.description="Network Observability EBPF Agent"
+LABEL summary="Network Observability EBPF Agent"
```

Comment:
nit: `eBPF` with low-case "e" is the conventional writing

```suggestion
LABEL io.k8s.display-name="Network Observability eBPF Agent"
LABEL io.k8s.description="Network Observability eBPF Agent"
LABEL summary="Network Observability eBPF Agent"
```

Reply:
Thanks!